DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningAssessmentViewModel.cs

```diff
@@ -29,10 +30,12 @@ public PostLearningAssessmentViewModel(PostLearningAssessment postLearningAssess
             if (postLearningAssessment.PostLearningAttempts == 0)
             {
                 AssessmentStatus = "Not attempted";
+                AssessmentStatusStyling = "not-passed-text";
             }
             else
             {
                 AssessmentStatus = GetPassStatus(postLearningAssessment);
+                AssessmentStatusStyling = GetPassStatusStyling(postLearningAssessment);
```

Comment:
You could assign this outside the if? Saves needing to type out the not passed css class twice (which means there's a chance they could become inconsistent). And I think assigning to a variable just once makes the code easier to read (instead of assigning to it in the if branch and the else) 🙂

Reply:
Yeah that looks better! Done in 42a3e2f 🙂